packages/cli/src/lib/pwa/index.ts

```diff
@@ -94,6 +96,15 @@ export class PWAGenerator {
     }
   }
 
+  private static compareManifest(manifest: Record<string, any>) {
```

Comment:
the func name compareXXX is confusing, accually it fill the defaults if not provide right?